2025-08-06 17:10:06,987 INFO    MainThread:7364 [wandb_setup.py:_flush():80] Current SDK version is 0.21.0
2025-08-06 17:10:06,987 INFO    MainThread:7364 [wandb_setup.py:_flush():80] Configure stats pid to 7364
2025-08-06 17:10:06,987 INFO    MainThread:7364 [wandb_setup.py:_flush():80] Loading settings from C:\Users\linus\.config\wandb\settings
2025-08-06 17:10:06,987 INFO    MainThread:7364 [wandb_setup.py:_flush():80] Loading settings from C:\Users\linus\Animal_Sound_Classification\wandb\settings
2025-08-06 17:10:06,987 INFO    MainThread:7364 [wandb_setup.py:_flush():80] Loading settings from environment variables
2025-08-06 17:10:06,987 INFO    MainThread:7364 [wandb_init.py:setup_run_log_directory():703] Logging user logs to C:\Users\linus\Animal_Sound_Classification\wandb\run-20250806_171006-34g86roi\logs\debug.log
2025-08-06 17:10:07,035 INFO    MainThread:7364 [wandb_init.py:setup_run_log_directory():704] Logging internal logs to C:\Users\linus\Animal_Sound_Classification\wandb\run-20250806_171006-34g86roi\logs\debug-internal.log
2025-08-06 17:10:07,035 INFO    MainThread:7364 [wandb_init.py:init():830] calling init triggers
2025-08-06 17:10:07,035 INFO    MainThread:7364 [wandb_init.py:init():835] wandb.init called with sweep_config: {}
config: {'epochs': 150, 'batch_size': 16, 'learning_rate': 0.001, '_wandb': {}}
2025-08-06 17:10:07,035 INFO    MainThread:7364 [wandb_init.py:init():871] starting backend
2025-08-06 17:10:07,595 INFO    MainThread:7364 [wandb_init.py:init():874] sending inform_init request
2025-08-06 17:10:07,619 INFO    MainThread:7364 [wandb_init.py:init():882] backend started and connected
2025-08-06 17:10:07,620 INFO    MainThread:7364 [wandb_init.py:init():953] updated telemetry
2025-08-06 17:10:07,708 INFO    MainThread:7364 [wandb_init.py:init():977] communicating run to backend with 90.0 second timeout
2025-08-06 17:10:08,544 INFO    MainThread:7364 [wandb_init.py:init():1029] starting run threads in backend
2025-08-06 17:10:08,816 INFO    MainThread:7364 [wandb_run.py:_console_start():2458] atexit reg
2025-08-06 17:10:08,816 INFO    MainThread:7364 [wandb_run.py:_redirect():2306] redirect: wrap_raw
2025-08-06 17:10:08,817 INFO    MainThread:7364 [wandb_run.py:_redirect():2375] Wrapping output streams.
2025-08-06 17:10:08,817 INFO    MainThread:7364 [wandb_run.py:_redirect():2398] Redirects installed.
2025-08-06 17:10:08,819 INFO    MainThread:7364 [wandb_init.py:init():1075] run started, returning control to user process
2025-08-06 17:10:08,821 INFO    MainThread:7364 [wandb_watch.py:_watch():70] Watching
2025-08-07 07:54:27,910 INFO    MsgRouterThr:7364 [mailbox.py:close():129] [no run ID] Closing mailbox, abandoning 1 handles.
2025-08-07 07:54:29,367 ERROR   MainThread:7364 [redirect.py:_on_write():664] [all runs] error in stderr callback
Traceback (most recent call last):
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\redirect.py", line 662, in _on_write
    cb(written_data)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 2385, in <lambda>
    lambda data: self._console_raw_callback("stderr", data),
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 398, in wrapper
    return func(self, *args, **kwargs)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 456, in wrapper_fn
    return func(self, *args, **kwargs)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 1566, in _console_raw_callback
    self._backend.interface.publish_output_raw(name, data)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\interface\interface.py", line 771, in publish_output_raw
    self._publish_output_raw(o)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\interface\interface_shared.py", line 38, in _publish_output_raw
    self._publish(rec)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\interface\interface_sock.py", line 39, in _publish
    self._sock_client.send_record_publish(record)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 170, in send_record_publish
    self.send_server_request(server_req)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 150, in send_server_request
    self._send_message(msg)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 147, in _send_message
    self._sendall_with_error_handle(header + data)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 126, in _sendall_with_error_handle
    sent = self._sock.send(data)
ConnectionAbortedError: [WinError 10053] An established connection was aborted by the software in your host machine
2025-08-07 07:54:29,379 ERROR   MainThread:7364 [redirect.py:_on_write():664] [all runs] error in stderr callback
Traceback (most recent call last):
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\redirect.py", line 662, in _on_write
    cb(written_data)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 2385, in <lambda>
    lambda data: self._console_raw_callback("stderr", data),
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 398, in wrapper
    return func(self, *args, **kwargs)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 456, in wrapper_fn
    return func(self, *args, **kwargs)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 1566, in _console_raw_callback
    self._backend.interface.publish_output_raw(name, data)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\interface\interface.py", line 771, in publish_output_raw
    self._publish_output_raw(o)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\interface\interface_shared.py", line 38, in _publish_output_raw
    self._publish(rec)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\interface\interface_sock.py", line 39, in _publish
    self._sock_client.send_record_publish(record)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 170, in send_record_publish
    self.send_server_request(server_req)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 150, in send_server_request
    self._send_message(msg)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 147, in _send_message
    self._sendall_with_error_handle(header + data)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 126, in _sendall_with_error_handle
    sent = self._sock.send(data)
ConnectionAbortedError: [WinError 10053] An established connection was aborted by the software in your host machine
2025-08-07 07:54:29,379 ERROR   MainThread:7364 [redirect.py:_on_write():664] [all runs] error in stderr callback
Traceback (most recent call last):
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\redirect.py", line 662, in _on_write
    cb(written_data)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 2385, in <lambda>
    lambda data: self._console_raw_callback("stderr", data),
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 398, in wrapper
    return func(self, *args, **kwargs)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 456, in wrapper_fn
    return func(self, *args, **kwargs)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 1566, in _console_raw_callback
    self._backend.interface.publish_output_raw(name, data)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\interface\interface.py", line 771, in publish_output_raw
    self._publish_output_raw(o)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\interface\interface_shared.py", line 38, in _publish_output_raw
    self._publish(rec)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\interface\interface_sock.py", line 39, in _publish
    self._sock_client.send_record_publish(record)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 170, in send_record_publish
    self.send_server_request(server_req)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 150, in send_server_request
    self._send_message(msg)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 147, in _send_message
    self._sendall_with_error_handle(header + data)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 126, in _sendall_with_error_handle
    sent = self._sock.send(data)
ConnectionAbortedError: [WinError 10053] An established connection was aborted by the software in your host machine
2025-08-07 07:54:29,380 ERROR   MainThread:7364 [redirect.py:_on_write():664] [all runs] error in stderr callback
Traceback (most recent call last):
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\redirect.py", line 662, in _on_write
    cb(written_data)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 2385, in <lambda>
    lambda data: self._console_raw_callback("stderr", data),
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 398, in wrapper
    return func(self, *args, **kwargs)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 456, in wrapper_fn
    return func(self, *args, **kwargs)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 1566, in _console_raw_callback
    self._backend.interface.publish_output_raw(name, data)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\interface\interface.py", line 771, in publish_output_raw
    self._publish_output_raw(o)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\interface\interface_shared.py", line 38, in _publish_output_raw
    self._publish(rec)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\interface\interface_sock.py", line 39, in _publish
    self._sock_client.send_record_publish(record)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 170, in send_record_publish
    self.send_server_request(server_req)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 150, in send_server_request
    self._send_message(msg)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 147, in _send_message
    self._sendall_with_error_handle(header + data)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 126, in _sendall_with_error_handle
    sent = self._sock.send(data)
ConnectionAbortedError: [WinError 10053] An established connection was aborted by the software in your host machine
2025-08-07 07:54:29,381 ERROR   MainThread:7364 [redirect.py:_on_write():664] [all runs] error in stderr callback
Traceback (most recent call last):
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\redirect.py", line 662, in _on_write
    cb(written_data)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 2385, in <lambda>
    lambda data: self._console_raw_callback("stderr", data),
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 398, in wrapper
    return func(self, *args, **kwargs)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 456, in wrapper_fn
    return func(self, *args, **kwargs)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 1566, in _console_raw_callback
    self._backend.interface.publish_output_raw(name, data)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\interface\interface.py", line 771, in publish_output_raw
    self._publish_output_raw(o)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\interface\interface_shared.py", line 38, in _publish_output_raw
    self._publish(rec)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\interface\interface_sock.py", line 39, in _publish
    self._sock_client.send_record_publish(record)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 170, in send_record_publish
    self.send_server_request(server_req)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 150, in send_server_request
    self._send_message(msg)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 147, in _send_message
    self._sendall_with_error_handle(header + data)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 126, in _sendall_with_error_handle
    sent = self._sock.send(data)
ConnectionAbortedError: [WinError 10053] An established connection was aborted by the software in your host machine
2025-08-07 07:54:29,384 ERROR   MainThread:7364 [redirect.py:_on_write():664] [all runs] error in stderr callback
Traceback (most recent call last):
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\redirect.py", line 662, in _on_write
    cb(written_data)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 2385, in <lambda>
    lambda data: self._console_raw_callback("stderr", data),
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 398, in wrapper
    return func(self, *args, **kwargs)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 456, in wrapper_fn
    return func(self, *args, **kwargs)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 1566, in _console_raw_callback
    self._backend.interface.publish_output_raw(name, data)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\interface\interface.py", line 771, in publish_output_raw
    self._publish_output_raw(o)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\interface\interface_shared.py", line 38, in _publish_output_raw
    self._publish(rec)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\interface\interface_sock.py", line 39, in _publish
    self._sock_client.send_record_publish(record)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 170, in send_record_publish
    self.send_server_request(server_req)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 150, in send_server_request
    self._send_message(msg)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 147, in _send_message
    self._sendall_with_error_handle(header + data)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 126, in _sendall_with_error_handle
    sent = self._sock.send(data)
ConnectionAbortedError: [WinError 10053] An established connection was aborted by the software in your host machine
2025-08-07 07:54:29,384 ERROR   MainThread:7364 [redirect.py:_on_write():664] [all runs] error in stderr callback
Traceback (most recent call last):
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\redirect.py", line 662, in _on_write
    cb(written_data)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 2385, in <lambda>
    lambda data: self._console_raw_callback("stderr", data),
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 398, in wrapper
    return func(self, *args, **kwargs)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 456, in wrapper_fn
    return func(self, *args, **kwargs)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 1566, in _console_raw_callback
    self._backend.interface.publish_output_raw(name, data)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\interface\interface.py", line 771, in publish_output_raw
    self._publish_output_raw(o)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\interface\interface_shared.py", line 38, in _publish_output_raw
    self._publish(rec)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\interface\interface_sock.py", line 39, in _publish
    self._sock_client.send_record_publish(record)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 170, in send_record_publish
    self.send_server_request(server_req)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 150, in send_server_request
    self._send_message(msg)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 147, in _send_message
    self._sendall_with_error_handle(header + data)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 126, in _sendall_with_error_handle
    sent = self._sock.send(data)
ConnectionAbortedError: [WinError 10053] An established connection was aborted by the software in your host machine
2025-08-07 07:54:29,385 ERROR   MainThread:7364 [redirect.py:_on_write():664] [all runs] error in stderr callback
Traceback (most recent call last):
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\redirect.py", line 662, in _on_write
    cb(written_data)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 2385, in <lambda>
    lambda data: self._console_raw_callback("stderr", data),
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 398, in wrapper
    return func(self, *args, **kwargs)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 456, in wrapper_fn
    return func(self, *args, **kwargs)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 1566, in _console_raw_callback
    self._backend.interface.publish_output_raw(name, data)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\interface\interface.py", line 771, in publish_output_raw
    self._publish_output_raw(o)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\interface\interface_shared.py", line 38, in _publish_output_raw
    self._publish(rec)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\interface\interface_sock.py", line 39, in _publish
    self._sock_client.send_record_publish(record)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 170, in send_record_publish
    self.send_server_request(server_req)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 150, in send_server_request
    self._send_message(msg)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 147, in _send_message
    self._sendall_with_error_handle(header + data)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 126, in _sendall_with_error_handle
    sent = self._sock.send(data)
ConnectionAbortedError: [WinError 10053] An established connection was aborted by the software in your host machine
2025-08-07 07:54:29,386 ERROR   MainThread:7364 [redirect.py:_on_write():664] [all runs] error in stderr callback
Traceback (most recent call last):
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\redirect.py", line 662, in _on_write
    cb(written_data)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 2385, in <lambda>
    lambda data: self._console_raw_callback("stderr", data),
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 398, in wrapper
    return func(self, *args, **kwargs)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 456, in wrapper_fn
    return func(self, *args, **kwargs)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 1566, in _console_raw_callback
    self._backend.interface.publish_output_raw(name, data)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\interface\interface.py", line 771, in publish_output_raw
    self._publish_output_raw(o)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\interface\interface_shared.py", line 38, in _publish_output_raw
    self._publish(rec)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\interface\interface_sock.py", line 39, in _publish
    self._sock_client.send_record_publish(record)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 170, in send_record_publish
    self.send_server_request(server_req)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 150, in send_server_request
    self._send_message(msg)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 147, in _send_message
    self._sendall_with_error_handle(header + data)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 126, in _sendall_with_error_handle
    sent = self._sock.send(data)
ConnectionAbortedError: [WinError 10053] An established connection was aborted by the software in your host machine
2025-08-07 07:54:29,387 ERROR   MainThread:7364 [redirect.py:_on_write():664] [all runs] error in stderr callback
Traceback (most recent call last):
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\redirect.py", line 662, in _on_write
    cb(written_data)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 2385, in <lambda>
    lambda data: self._console_raw_callback("stderr", data),
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 398, in wrapper
    return func(self, *args, **kwargs)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 456, in wrapper_fn
    return func(self, *args, **kwargs)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 1566, in _console_raw_callback
    self._backend.interface.publish_output_raw(name, data)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\interface\interface.py", line 771, in publish_output_raw
    self._publish_output_raw(o)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\interface\interface_shared.py", line 38, in _publish_output_raw
    self._publish(rec)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\interface\interface_sock.py", line 39, in _publish
    self._sock_client.send_record_publish(record)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 170, in send_record_publish
    self.send_server_request(server_req)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 150, in send_server_request
    self._send_message(msg)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 147, in _send_message
    self._sendall_with_error_handle(header + data)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 126, in _sendall_with_error_handle
    sent = self._sock.send(data)
ConnectionAbortedError: [WinError 10053] An established connection was aborted by the software in your host machine
2025-08-07 07:54:29,387 ERROR   MainThread:7364 [redirect.py:_on_write():664] [all runs] error in stderr callback
Traceback (most recent call last):
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\redirect.py", line 662, in _on_write
    cb(written_data)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 2385, in <lambda>
    lambda data: self._console_raw_callback("stderr", data),
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 398, in wrapper
    return func(self, *args, **kwargs)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 456, in wrapper_fn
    return func(self, *args, **kwargs)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 1566, in _console_raw_callback
    self._backend.interface.publish_output_raw(name, data)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\interface\interface.py", line 771, in publish_output_raw
    self._publish_output_raw(o)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\interface\interface_shared.py", line 38, in _publish_output_raw
    self._publish(rec)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\interface\interface_sock.py", line 39, in _publish
    self._sock_client.send_record_publish(record)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 170, in send_record_publish
    self.send_server_request(server_req)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 150, in send_server_request
    self._send_message(msg)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 147, in _send_message
    self._sendall_with_error_handle(header + data)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 126, in _sendall_with_error_handle
    sent = self._sock.send(data)
ConnectionAbortedError: [WinError 10053] An established connection was aborted by the software in your host machine
2025-08-07 07:54:29,388 ERROR   MainThread:7364 [redirect.py:_on_write():664] [all runs] error in stderr callback
Traceback (most recent call last):
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\redirect.py", line 662, in _on_write
    cb(written_data)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 2385, in <lambda>
    lambda data: self._console_raw_callback("stderr", data),
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 398, in wrapper
    return func(self, *args, **kwargs)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 456, in wrapper_fn
    return func(self, *args, **kwargs)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 1566, in _console_raw_callback
    self._backend.interface.publish_output_raw(name, data)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\interface\interface.py", line 771, in publish_output_raw
    self._publish_output_raw(o)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\interface\interface_shared.py", line 38, in _publish_output_raw
    self._publish(rec)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\interface\interface_sock.py", line 39, in _publish
    self._sock_client.send_record_publish(record)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 170, in send_record_publish
    self.send_server_request(server_req)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 150, in send_server_request
    self._send_message(msg)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 147, in _send_message
    self._sendall_with_error_handle(header + data)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 126, in _sendall_with_error_handle
    sent = self._sock.send(data)
ConnectionAbortedError: [WinError 10053] An established connection was aborted by the software in your host machine
2025-08-07 07:54:29,388 ERROR   MainThread:7364 [redirect.py:_on_write():664] [all runs] error in stderr callback
Traceback (most recent call last):
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\redirect.py", line 662, in _on_write
    cb(written_data)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 2385, in <lambda>
    lambda data: self._console_raw_callback("stderr", data),
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 398, in wrapper
    return func(self, *args, **kwargs)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 456, in wrapper_fn
    return func(self, *args, **kwargs)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 1566, in _console_raw_callback
    self._backend.interface.publish_output_raw(name, data)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\interface\interface.py", line 771, in publish_output_raw
    self._publish_output_raw(o)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\interface\interface_shared.py", line 38, in _publish_output_raw
    self._publish(rec)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\interface\interface_sock.py", line 39, in _publish
    self._sock_client.send_record_publish(record)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 170, in send_record_publish
    self.send_server_request(server_req)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 150, in send_server_request
    self._send_message(msg)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 147, in _send_message
    self._sendall_with_error_handle(header + data)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 126, in _sendall_with_error_handle
    sent = self._sock.send(data)
ConnectionAbortedError: [WinError 10053] An established connection was aborted by the software in your host machine
2025-08-07 07:54:29,391 ERROR   MainThread:7364 [redirect.py:_on_write():664] [all runs] error in stderr callback
Traceback (most recent call last):
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\redirect.py", line 662, in _on_write
    cb(written_data)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 2385, in <lambda>
    lambda data: self._console_raw_callback("stderr", data),
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 398, in wrapper
    return func(self, *args, **kwargs)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 456, in wrapper_fn
    return func(self, *args, **kwargs)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 1566, in _console_raw_callback
    self._backend.interface.publish_output_raw(name, data)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\interface\interface.py", line 771, in publish_output_raw
    self._publish_output_raw(o)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\interface\interface_shared.py", line 38, in _publish_output_raw
    self._publish(rec)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\interface\interface_sock.py", line 39, in _publish
    self._sock_client.send_record_publish(record)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 170, in send_record_publish
    self.send_server_request(server_req)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 150, in send_server_request
    self._send_message(msg)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 147, in _send_message
    self._sendall_with_error_handle(header + data)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 126, in _sendall_with_error_handle
    sent = self._sock.send(data)
ConnectionAbortedError: [WinError 10053] An established connection was aborted by the software in your host machine
2025-08-07 07:54:29,392 ERROR   MainThread:7364 [redirect.py:_on_write():664] [all runs] error in stderr callback
Traceback (most recent call last):
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\redirect.py", line 662, in _on_write
    cb(written_data)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 2385, in <lambda>
    lambda data: self._console_raw_callback("stderr", data),
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 398, in wrapper
    return func(self, *args, **kwargs)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 456, in wrapper_fn
    return func(self, *args, **kwargs)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 1566, in _console_raw_callback
    self._backend.interface.publish_output_raw(name, data)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\interface\interface.py", line 771, in publish_output_raw
    self._publish_output_raw(o)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\interface\interface_shared.py", line 38, in _publish_output_raw
    self._publish(rec)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\interface\interface_sock.py", line 39, in _publish
    self._sock_client.send_record_publish(record)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 170, in send_record_publish
    self.send_server_request(server_req)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 150, in send_server_request
    self._send_message(msg)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 147, in _send_message
    self._sendall_with_error_handle(header + data)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 126, in _sendall_with_error_handle
    sent = self._sock.send(data)
ConnectionAbortedError: [WinError 10053] An established connection was aborted by the software in your host machine
2025-08-07 07:54:29,392 ERROR   MainThread:7364 [redirect.py:_on_write():664] [all runs] error in stderr callback
Traceback (most recent call last):
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\redirect.py", line 662, in _on_write
    cb(written_data)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 2385, in <lambda>
    lambda data: self._console_raw_callback("stderr", data),
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 398, in wrapper
    return func(self, *args, **kwargs)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 456, in wrapper_fn
    return func(self, *args, **kwargs)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 1566, in _console_raw_callback
    self._backend.interface.publish_output_raw(name, data)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\interface\interface.py", line 771, in publish_output_raw
    self._publish_output_raw(o)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\interface\interface_shared.py", line 38, in _publish_output_raw
    self._publish(rec)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\interface\interface_sock.py", line 39, in _publish
    self._sock_client.send_record_publish(record)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 170, in send_record_publish
    self.send_server_request(server_req)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 150, in send_server_request
    self._send_message(msg)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 147, in _send_message
    self._sendall_with_error_handle(header + data)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 126, in _sendall_with_error_handle
    sent = self._sock.send(data)
ConnectionAbortedError: [WinError 10053] An established connection was aborted by the software in your host machine
2025-08-07 07:54:29,393 ERROR   MainThread:7364 [redirect.py:_on_write():664] [all runs] error in stderr callback
Traceback (most recent call last):
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\redirect.py", line 662, in _on_write
    cb(written_data)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 2385, in <lambda>
    lambda data: self._console_raw_callback("stderr", data),
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 398, in wrapper
    return func(self, *args, **kwargs)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 456, in wrapper_fn
    return func(self, *args, **kwargs)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 1566, in _console_raw_callback
    self._backend.interface.publish_output_raw(name, data)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\interface\interface.py", line 771, in publish_output_raw
    self._publish_output_raw(o)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\interface\interface_shared.py", line 38, in _publish_output_raw
    self._publish(rec)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\interface\interface_sock.py", line 39, in _publish
    self._sock_client.send_record_publish(record)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 170, in send_record_publish
    self.send_server_request(server_req)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 150, in send_server_request
    self._send_message(msg)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 147, in _send_message
    self._sendall_with_error_handle(header + data)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 126, in _sendall_with_error_handle
    sent = self._sock.send(data)
ConnectionAbortedError: [WinError 10053] An established connection was aborted by the software in your host machine
2025-08-07 07:54:29,396 ERROR   MainThread:7364 [redirect.py:_on_write():664] [all runs] error in stderr callback
Traceback (most recent call last):
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\redirect.py", line 662, in _on_write
    cb(written_data)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 2385, in <lambda>
    lambda data: self._console_raw_callback("stderr", data),
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 398, in wrapper
    return func(self, *args, **kwargs)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 456, in wrapper_fn
    return func(self, *args, **kwargs)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 1566, in _console_raw_callback
    self._backend.interface.publish_output_raw(name, data)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\interface\interface.py", line 771, in publish_output_raw
    self._publish_output_raw(o)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\interface\interface_shared.py", line 38, in _publish_output_raw
    self._publish(rec)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\interface\interface_sock.py", line 39, in _publish
    self._sock_client.send_record_publish(record)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 170, in send_record_publish
    self.send_server_request(server_req)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 150, in send_server_request
    self._send_message(msg)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 147, in _send_message
    self._sendall_with_error_handle(header + data)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 126, in _sendall_with_error_handle
    sent = self._sock.send(data)
ConnectionAbortedError: [WinError 10053] An established connection was aborted by the software in your host machine
2025-08-07 07:54:29,397 ERROR   MainThread:7364 [redirect.py:_on_write():664] [all runs] error in stderr callback
Traceback (most recent call last):
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\redirect.py", line 662, in _on_write
    cb(written_data)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 2385, in <lambda>
    lambda data: self._console_raw_callback("stderr", data),
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 398, in wrapper
    return func(self, *args, **kwargs)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 456, in wrapper_fn
    return func(self, *args, **kwargs)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 1566, in _console_raw_callback
    self._backend.interface.publish_output_raw(name, data)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\interface\interface.py", line 771, in publish_output_raw
    self._publish_output_raw(o)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\interface\interface_shared.py", line 38, in _publish_output_raw
    self._publish(rec)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\interface\interface_sock.py", line 39, in _publish
    self._sock_client.send_record_publish(record)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 170, in send_record_publish
    self.send_server_request(server_req)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 150, in send_server_request
    self._send_message(msg)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 147, in _send_message
    self._sendall_with_error_handle(header + data)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 126, in _sendall_with_error_handle
    sent = self._sock.send(data)
ConnectionAbortedError: [WinError 10053] An established connection was aborted by the software in your host machine
2025-08-07 07:54:29,397 ERROR   MainThread:7364 [redirect.py:_on_write():664] [all runs] error in stderr callback
Traceback (most recent call last):
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\redirect.py", line 662, in _on_write
    cb(written_data)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 2385, in <lambda>
    lambda data: self._console_raw_callback("stderr", data),
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 398, in wrapper
    return func(self, *args, **kwargs)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 456, in wrapper_fn
    return func(self, *args, **kwargs)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 1566, in _console_raw_callback
    self._backend.interface.publish_output_raw(name, data)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\interface\interface.py", line 771, in publish_output_raw
    self._publish_output_raw(o)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\interface\interface_shared.py", line 38, in _publish_output_raw
    self._publish(rec)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\interface\interface_sock.py", line 39, in _publish
    self._sock_client.send_record_publish(record)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 170, in send_record_publish
    self.send_server_request(server_req)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 150, in send_server_request
    self._send_message(msg)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 147, in _send_message
    self._sendall_with_error_handle(header + data)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 126, in _sendall_with_error_handle
    sent = self._sock.send(data)
ConnectionAbortedError: [WinError 10053] An established connection was aborted by the software in your host machine
2025-08-07 07:54:29,398 ERROR   MainThread:7364 [redirect.py:_on_write():664] [all runs] error in stderr callback
Traceback (most recent call last):
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\redirect.py", line 662, in _on_write
    cb(written_data)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 2385, in <lambda>
    lambda data: self._console_raw_callback("stderr", data),
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 398, in wrapper
    return func(self, *args, **kwargs)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 456, in wrapper_fn
    return func(self, *args, **kwargs)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 1566, in _console_raw_callback
    self._backend.interface.publish_output_raw(name, data)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\interface\interface.py", line 771, in publish_output_raw
    self._publish_output_raw(o)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\interface\interface_shared.py", line 38, in _publish_output_raw
    self._publish(rec)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\interface\interface_sock.py", line 39, in _publish
    self._sock_client.send_record_publish(record)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 170, in send_record_publish
    self.send_server_request(server_req)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 150, in send_server_request
    self._send_message(msg)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 147, in _send_message
    self._sendall_with_error_handle(header + data)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 126, in _sendall_with_error_handle
    sent = self._sock.send(data)
ConnectionAbortedError: [WinError 10053] An established connection was aborted by the software in your host machine
2025-08-07 07:54:29,399 ERROR   MainThread:7364 [redirect.py:_on_write():664] [all runs] error in stderr callback
Traceback (most recent call last):
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\redirect.py", line 662, in _on_write
    cb(written_data)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 2385, in <lambda>
    lambda data: self._console_raw_callback("stderr", data),
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 398, in wrapper
    return func(self, *args, **kwargs)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 456, in wrapper_fn
    return func(self, *args, **kwargs)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 1566, in _console_raw_callback
    self._backend.interface.publish_output_raw(name, data)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\interface\interface.py", line 771, in publish_output_raw
    self._publish_output_raw(o)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\interface\interface_shared.py", line 38, in _publish_output_raw
    self._publish(rec)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\interface\interface_sock.py", line 39, in _publish
    self._sock_client.send_record_publish(record)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 170, in send_record_publish
    self.send_server_request(server_req)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 150, in send_server_request
    self._send_message(msg)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 147, in _send_message
    self._sendall_with_error_handle(header + data)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 126, in _sendall_with_error_handle
    sent = self._sock.send(data)
ConnectionAbortedError: [WinError 10053] An established connection was aborted by the software in your host machine
2025-08-07 07:54:29,401 ERROR   MainThread:7364 [redirect.py:_on_write():664] [all runs] error in stderr callback
Traceback (most recent call last):
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\redirect.py", line 662, in _on_write
    cb(written_data)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 2385, in <lambda>
    lambda data: self._console_raw_callback("stderr", data),
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 398, in wrapper
    return func(self, *args, **kwargs)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 456, in wrapper_fn
    return func(self, *args, **kwargs)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 1566, in _console_raw_callback
    self._backend.interface.publish_output_raw(name, data)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\interface\interface.py", line 771, in publish_output_raw
    self._publish_output_raw(o)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\interface\interface_shared.py", line 38, in _publish_output_raw
    self._publish(rec)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\interface\interface_sock.py", line 39, in _publish
    self._sock_client.send_record_publish(record)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 170, in send_record_publish
    self.send_server_request(server_req)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 150, in send_server_request
    self._send_message(msg)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 147, in _send_message
    self._sendall_with_error_handle(header + data)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 126, in _sendall_with_error_handle
    sent = self._sock.send(data)
ConnectionAbortedError: [WinError 10053] An established connection was aborted by the software in your host machine
2025-08-07 07:54:29,401 ERROR   MainThread:7364 [redirect.py:_on_write():664] [all runs] error in stderr callback
Traceback (most recent call last):
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\redirect.py", line 662, in _on_write
    cb(written_data)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 2385, in <lambda>
    lambda data: self._console_raw_callback("stderr", data),
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 398, in wrapper
    return func(self, *args, **kwargs)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 456, in wrapper_fn
    return func(self, *args, **kwargs)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 1566, in _console_raw_callback
    self._backend.interface.publish_output_raw(name, data)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\interface\interface.py", line 771, in publish_output_raw
    self._publish_output_raw(o)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\interface\interface_shared.py", line 38, in _publish_output_raw
    self._publish(rec)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\interface\interface_sock.py", line 39, in _publish
    self._sock_client.send_record_publish(record)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 170, in send_record_publish
    self.send_server_request(server_req)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 150, in send_server_request
    self._send_message(msg)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 147, in _send_message
    self._sendall_with_error_handle(header + data)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 126, in _sendall_with_error_handle
    sent = self._sock.send(data)
ConnectionAbortedError: [WinError 10053] An established connection was aborted by the software in your host machine
2025-08-07 07:54:29,402 ERROR   MainThread:7364 [redirect.py:_on_write():664] [all runs] error in stderr callback
Traceback (most recent call last):
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\redirect.py", line 662, in _on_write
    cb(written_data)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 2385, in <lambda>
    lambda data: self._console_raw_callback("stderr", data),
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 398, in wrapper
    return func(self, *args, **kwargs)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 456, in wrapper_fn
    return func(self, *args, **kwargs)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 1566, in _console_raw_callback
    self._backend.interface.publish_output_raw(name, data)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\interface\interface.py", line 771, in publish_output_raw
    self._publish_output_raw(o)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\interface\interface_shared.py", line 38, in _publish_output_raw
    self._publish(rec)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\interface\interface_sock.py", line 39, in _publish
    self._sock_client.send_record_publish(record)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 170, in send_record_publish
    self.send_server_request(server_req)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 150, in send_server_request
    self._send_message(msg)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 147, in _send_message
    self._sendall_with_error_handle(header + data)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 126, in _sendall_with_error_handle
    sent = self._sock.send(data)
ConnectionAbortedError: [WinError 10053] An established connection was aborted by the software in your host machine
2025-08-07 07:54:29,402 ERROR   MainThread:7364 [redirect.py:_on_write():664] [all runs] error in stderr callback
Traceback (most recent call last):
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\redirect.py", line 662, in _on_write
    cb(written_data)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 2385, in <lambda>
    lambda data: self._console_raw_callback("stderr", data),
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 398, in wrapper
    return func(self, *args, **kwargs)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 456, in wrapper_fn
    return func(self, *args, **kwargs)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 1566, in _console_raw_callback
    self._backend.interface.publish_output_raw(name, data)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\interface\interface.py", line 771, in publish_output_raw
    self._publish_output_raw(o)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\interface\interface_shared.py", line 38, in _publish_output_raw
    self._publish(rec)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\interface\interface_sock.py", line 39, in _publish
    self._sock_client.send_record_publish(record)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 170, in send_record_publish
    self.send_server_request(server_req)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 150, in send_server_request
    self._send_message(msg)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 147, in _send_message
    self._sendall_with_error_handle(header + data)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 126, in _sendall_with_error_handle
    sent = self._sock.send(data)
ConnectionAbortedError: [WinError 10053] An established connection was aborted by the software in your host machine
2025-08-07 07:54:29,403 ERROR   MainThread:7364 [redirect.py:_on_write():664] [all runs] error in stderr callback
Traceback (most recent call last):
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\redirect.py", line 662, in _on_write
    cb(written_data)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 2385, in <lambda>
    lambda data: self._console_raw_callback("stderr", data),
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 398, in wrapper
    return func(self, *args, **kwargs)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 456, in wrapper_fn
    return func(self, *args, **kwargs)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 1566, in _console_raw_callback
    self._backend.interface.publish_output_raw(name, data)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\interface\interface.py", line 771, in publish_output_raw
    self._publish_output_raw(o)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\interface\interface_shared.py", line 38, in _publish_output_raw
    self._publish(rec)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\interface\interface_sock.py", line 39, in _publish
    self._sock_client.send_record_publish(record)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 170, in send_record_publish
    self.send_server_request(server_req)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 150, in send_server_request
    self._send_message(msg)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 147, in _send_message
    self._sendall_with_error_handle(header + data)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 126, in _sendall_with_error_handle
    sent = self._sock.send(data)
ConnectionAbortedError: [WinError 10053] An established connection was aborted by the software in your host machine
2025-08-07 07:54:29,403 ERROR   MainThread:7364 [redirect.py:_on_write():664] [all runs] error in stderr callback
Traceback (most recent call last):
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\redirect.py", line 662, in _on_write
    cb(written_data)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 2385, in <lambda>
    lambda data: self._console_raw_callback("stderr", data),
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 398, in wrapper
    return func(self, *args, **kwargs)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 456, in wrapper_fn
    return func(self, *args, **kwargs)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 1566, in _console_raw_callback
    self._backend.interface.publish_output_raw(name, data)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\interface\interface.py", line 771, in publish_output_raw
    self._publish_output_raw(o)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\interface\interface_shared.py", line 38, in _publish_output_raw
    self._publish(rec)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\interface\interface_sock.py", line 39, in _publish
    self._sock_client.send_record_publish(record)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 170, in send_record_publish
    self.send_server_request(server_req)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 150, in send_server_request
    self._send_message(msg)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 147, in _send_message
    self._sendall_with_error_handle(header + data)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 126, in _sendall_with_error_handle
    sent = self._sock.send(data)
ConnectionAbortedError: [WinError 10053] An established connection was aborted by the software in your host machine
2025-08-07 07:54:29,546 ERROR   MainThread:7364 [redirect.py:_on_write():664] [all runs] error in stdout callback
Traceback (most recent call last):
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\redirect.py", line 662, in _on_write
    cb(written_data)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 2379, in <lambda>
    lambda data: self._console_raw_callback("stdout", data),
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 398, in wrapper
    return func(self, *args, **kwargs)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 456, in wrapper_fn
    return func(self, *args, **kwargs)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\wandb_run.py", line 1566, in _console_raw_callback
    self._backend.interface.publish_output_raw(name, data)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\interface\interface.py", line 771, in publish_output_raw
    self._publish_output_raw(o)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\interface\interface_shared.py", line 38, in _publish_output_raw
    self._publish(rec)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\interface\interface_sock.py", line 39, in _publish
    self._sock_client.send_record_publish(record)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 170, in send_record_publish
    self.send_server_request(server_req)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 150, in send_server_request
    self._send_message(msg)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 147, in _send_message
    self._sendall_with_error_handle(header + data)
  File "C:\Users\linus\miniconda3\envs\animal-sound-ml\lib\site-packages\wandb\sdk\lib\sock_client.py", line 126, in _sendall_with_error_handle
    sent = self._sock.send(data)
ConnectionAbortedError: [WinError 10053] An established connection was aborted by the software in your host machine
